pkg/internal/apis/certmanager/validation/certificate.go

```diff
@@ -79,6 +79,9 @@ func ValidateCertificateSpec(crt *internalcmapi.CertificateSpec, fldPath *field.
 	if len(crt.Usages) > 0 {
 		el = append(el, validateUsages(crt, fldPath)...)
 	}
+	if crt.RevisionHistoryLimit != nil && *crt.RevisionHistoryLimit < 1 {
+		el = append(el, field.Invalid(fldPath.Child("revisionHistoryLimit"), crt.RevisionHistoryLimit, "must not be less than 1"))
```

Comment:
`		el = append(el, field.Invalid(fldPath.Child("revisionHistoryLimit"), *crt.RevisionHistoryLimit, "must not be less than 1"))`